integreat_cms/cms/constants/roles.py

```diff
@@ -75,6 +75,8 @@
     "change_imprintpage",
     "change_pushnotification",
     "change_user",
+    "change_chatmessage",
+    "delete_chatmessage",
```

Comment:
```suggestion
```
I don't see a reason why we should allow the management role to delete all chat messages (including those by other users)... they are already allowed to delete their own messages...